packages/esm-patient-chart-app/src/root.component.tsx

```diff
@@ -6,14 +6,22 @@ import { BrowserRouter, Route } from 'react-router-dom';
 import { basePath, dashboardPath, spaRoot } from './constants';
 import styles from './root.scss';
 import { ContextWindowSizeProvider } from './hooks/useContextWindowSize';
+import { SWRConfig } from 'swr';
+
+const swrConfiguration = {
+  // Maximum number of retries when the backend returns an error
+  errorRetryCount: 3,
+};
 
 export default function Root() {
   return (
     <BrowserRouter basename={spaRoot}>
       <ContextWindowSizeProvider>
         <div className={styles.patientChartWrapper}>
           <SideMenu />
-          <Route path={dashboardPath} component={PatientChart} />
+          <SWRConfig value={{ ...swrConfiguration }}>
```

Comment:
Any reason for re-creating the object? Would `value={swrConfiguration}` be sufficient?

The other question is why we only wrap one route - would it make sense to wrap the whole patient chart wrapper (or even the router) in the config?

Reply:
1. Oops.
2. Fair enough